### PR TITLE
Updated Tatar Translation!

### PR DESCRIPTION
**Pull request by Amirhan-Taipovjan-Greatest-I**

### CODE REVIEW

**Comment by TheCSDev** [comment]
It appears I have completely forgotten about this pull request. And as time passed, the translation keys and files have changed a bit, resulting in conflicts.
If you don't mind, you may feel free to close this request (can't be merged anyways due to conflicts), and open a new and updated one.

Thanks tho, and have a great day!

**Comment by Amirhan-Taipovjan-Greatest-I** [comment]
> It appears I have completely forgotten about this pull request. And as time passed, the translation keys and files have changed a bit, resulting in conflicts. If you don't mind, you may feel free to close this request (can't be merged anyways due to conflicts), and open a new and updated one.
> 
> Thanks tho, and have a great day!

Is "Featured" word about people or other projects?

**Comment by TheCSDev** [comment]
Hello. The word "Featured" is meant to be a general word for "Featured entries in a list". Currently, it is used in `betterstats` to show a list of "featured people" in the "Credits" tab. 

However, although I am not sure, there may be a chance I end up using that word for "featured lists of projects" too. Let me know if that's an issue. If it is, I'll end up creating a separate key for "featured projects" in the future if I end up needing it.

TLDR; Altho it's meant to be a general term for both, right now it is only used for lists of people.

**Comment by TheCSDev** [comment]
I took another look at my translation file, and this is how it appears I decided to format it:
```json
"tcdcommons.featured.no_one": "No one is on this list at the moment."
```
So, given that for "people", I used `no_one`, if I ever end up needing one for projects, I'll just create another key called `tcdcommons.featured.no_projects`.

Just thought I'd let you know.
